packages/cli-repl/src/completer.spec.ts

```diff
@@ -1,5 +1,5 @@
 import completer from './completer';
-import { types as shellTypes } from '@mongosh/shell-api';
```

Comment:
I think my editor made this change automatically, can revert if anyone sees a problem with it.